android/Hybridge/src/com/pdi/hybridge/HybridgeWebChromeClient.java

```diff
@@ -32,7 +32,9 @@
     public HybridgeWebChromeClient(JsAction[] actions) {
         mActions = new HashMap<String, Class>(actions.length);
         for (final JsAction action : actions) {
-            mActions.put(action.toString().toLowerCase(), action.getTask());
+            if (action != null) {
```

Comment:
This should fail, is not possible to call any action not listed from native at init, Otherwise is a fake action and it should fail early.